src/lib.rs

```diff
@@ -1,3 +1,386 @@
+//!<p align="center"><img src="https://raw.githubusercontent.com/ayrat555/fang/master/logo.png" alt="fang" height="300px"></p>
```

Comment:
would it be possible to just copy from readme using `include_str`?

[#![doc = include_str!("README.md")]
](https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1540-2021-07-29)

Reply:
Done :D [README](https://github.com/ayrat555/fang/pull/80/commits/79b457f3798e07f651cb445a3adf5f6f2b0906da)